judge/models/contest.py

```diff
@@ -113,6 +115,10 @@ class Contest(models.Model):
                               help_text=_('A JSON object to serve as the configuration for the chosen contest format '
                                           'module. Leave empty to use None. Exact format depends on the contest format '
                                           'selected.'))
+    problem_label_script = models.TextField(verbose_name='contest problem label script', blank=True,
+                                            help_text='A custom LUA function to generate problem labels. Requires a '
+                                                      'single function with an integer paramater, the contest problem '
+                                                      '"order", and returns a string, the label.')
```

Comment:
0-indexed or 1-indexed?

Reply:
This is dependant on the contest setter (if labels are generated with `problem.order`).